FogCreek/Glitch-Community
# Turquoise beetle: Update Collection pages to properly render "My Stuff"

## Links
* https://turquoise-beetle.glitch.me
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/169
* https://www.notion.so/glitch/My-Stuff-Default-Collection-f9eb3c8196c6431596fba526fc5b91c2#d6cbf32aa4c542d7a0decd0016b811c8

## GIF/Screenshots:
<img width="1011" alt="Screen Shot 2019-07-30 at 3 59 15 PM" src="https://user-images.githubusercontent.com/6620164/62161101-6e569880-b2e3-11e9-908c-da09c98ccda8.png">

## Changes:
On collection pages when on a my stuff collection and the feature is enabled:
- shows the right avatar
- disable editing name/description
- removes delete button

## How To Test:
* enable my stuff on the secret page
* create a my stuff collection by clicking on it on your user page and adding a project to it
* notice that the page renders the right avatar and disables editing name/description and deleting. You should still be able to change color and add/remove projects

## Feedback I'm looking for:
bugs?
